content/en/docs/tutorials/security/apparmor.md

```diff
@@ -70,11 +64,11 @@ Make sure:
    If the Kubelet contains AppArmor support (>= v1.4), it will refuse to run a Pod with AppArmor
    options if the kernel module is not enabled.
 
-  {{< note >}}
-  Ubuntu carries many AppArmor patches that have not been merged into the upstream Linux
-  kernel, including patches that add additional hooks and features. Kubernetes has only been
-  tested with the upstream version, and does not promise support for other features.
-  {{< /note >}}
+{{< note >}}
```

Comment:
The original indentation is correct. See the [current online page](https://kubernetes.io/docs/tutorials/security/apparmor/#before-you-begin).